Description: Don't set X-GNOME-Autostart-Phase to not delay other software start during GNOME login
Author: ?

diff -Nur gnome-session-2.28.0/data/gnome-wm.desktop.in.in gnome-session-2.28.0.ubuntu/data/gnome-wm.desktop.in.in
--- gnome-session-2.28.0/data/gnome-wm.desktop.in.in	2009-08-25 10:03:42.000000000 +0200
+++ gnome-session-2.28.0.ubuntu/data/gnome-wm.desktop.in.in	2010-01-20 22:21:19.000000000 +0100
@@ -3,7 +3,7 @@
 _Name=Window Manager
 Exec=gnome-wm
 NoDisplay=true
-X-GNOME-Autostart-Phase=WindowManager
+#X-GNOME-Autostart-Phase=WindowManager
 X-GNOME-Provides=windowmanager
 X-GNOME-Autostart-Notify=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
